setup.py

```diff
@@ -26,6 +26,8 @@
 
 # Configure the required packages and scripts to install.
 with open('requirements.txt', encoding='utf-8') as requirements_txt: # pylint: disable=replace-disallowed-function-calls
+    # The 'parse_requirements' returns list of 'Requirement' objects and we need
```

Comment:
list --> a list

"objects and we need..." --> "objects. We need to transform these to strings using the `str()` function."

Reply:
Done.